addons/l10n_br_website_sale/controllers/main.py

```diff
@@ -39,15 +39,17 @@ def _prepare_address_form_values(self, order_sudo, partner_sudo, *args, address_
         rendering_values = super()._prepare_address_form_values(
             order_sudo, partner_sudo, *args, address_type=address_type, **kwargs
         )
-        if (kwargs.get('use_delivery_as_billing') and address_type == 'delivery' or address_type == 'billing') and request.website.sudo().company_id.account_fiscal_country_id.code == 'BR':
-            can_edit_vat = rendering_values['can_edit_vat']
-            LatamIdentificationType = request.env['l10n_latam.identification.type'].sudo()
-            rendering_values.update({
-                'identification_types': LatamIdentificationType.search([
-                    '|', ('country_id', '=', False), ('country_id.code', '=', 'BR'),
-                ]) if can_edit_vat else LatamIdentificationType,
-            })
+        if request.website.sudo().company_id.account_fiscal_country_id.code == 'BR':
             rendering_values['city_sudo'] = partner_sudo.city_id
             rendering_values['cities_sudo'] = request.env['res.city'].sudo().search([('country_id.code', '=', 'BR')])
-            rendering_values['vat_label'] = _lt('Number')
+
+            if kwargs.get('use_delivery_as_billing') and address_type == 'delivery' or address_type == 'billing':
```

Comment:
```suggestion
            if (kwargs.get('use_delivery_as_billing') and address_type == 'delivery') or address_type == 'billing':
```
I can never remember whether `and` or `or` has precendence and have to look it up. Let's make it explicit while we're changing these lines.